openhcl/virt_mshv_vtl/src/lib.rs

```diff
@@ -24,6 +24,9 @@ cfg_if::cfg_if!(
         use processor::tdx::TdxBackedShared;
         use std::arch::x86_64::CpuidResult;
         use virt::CpuidLeaf;
+        use bitvec::prelude::BitArray;
+        use bitvec::prelude::Lsb0;
+        type IrrBitmap = BitArray<[u32; 8], Lsb0>;
```

Comment:
could you leave a comment on what this type is? something like `/// Bitarray type for representing IRR bits in a x86-64 APIC` or such